.github/workflows/release_gem.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Release
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    tags:
+      - '**'
+
+jobs:
+  release:
+    name: Release gem
+    uses: theforeman/actions/.github/workflows/release-gem.yml@v0
+    with:
+      allowed_owner: theforeman
+    secrets:
+      api_key: ${{ secrets.RUBYGEM_API_KEY }}
```

Comment:
Pardon I'm just unsure here: where is this secret coming from? Is that organizational secret? :)

Reply:
It is not (we have per-repo secrets that can only push that one gem)
I'll create you one!

Reply:
done

Reply:
Thanks!